### PR TITLE
iOS feature：Hardware Bluetooth low version compatible

### PR DESCRIPTION
**Pull request by DevoutCoder**
## What does this implement/fix? Explain your changes.
iOS feature：Hardware Bluetooth low version compatible

## Pull request type
 
_Put an `x` in the boxes that apply_
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: ios/Support/OneKey-Info.plist [hunk 1]
```diff
@@ -32,7 +32,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>2.0.8</string>
+	<string>$(MARKETING_VERSION)</string>
```
Comment (danielsocials) [inline ios/Support/OneKey-Info.plist:35]: MARKETING_VERSION 这个在哪里定义的，环境变量？

File: ios/CustomCode/Modules/Manager/OKUserSettingManager.m [hunk 2]
```diff
@@ -119,7 +119,8 @@ - (void)setPinInputMethod:(OKDevicePINInputMethod)pinInputMethod {
 }
 
 - (OKDevicePINInputMethod)pinInputMethod {
-    return [[OKStorageManager loadFromUserDefaults:kPinInputMethod] integerValue];
+    return OKDevicePINInputMethodOnApp;
+//    return [[OKStorageManager loadFromUserDefaults:kPinInputMethod] integerValue];
```
Comment (danielsocials) [inline ios/CustomCode/Modules/Manager/OKUserSettingManager.m:123]: 删除无用代码

Comment (DevoutCoder) [reply ios/CustomCode/Modules/Manager/OKUserSettingManager.m]: 已改

File: ios/Support/OneKey-Info.plist (hunk 1 above)
Comment (DevoutCoder) [reply ios/Support/OneKey-Info.plist]: 已改

**Review by danielsocials — APPROVED** [review]

**Review by liuzjalex — APPROVED** [review]

**Pull request merged**